packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -926,7 +927,7 @@ export class Stack extends Construct implements ITaggable {
    * - Don't forget to remove the `exportValue()` call as well.
    * - Deploy again (this time only the `producerStack` will be changed -- the bucket will be deleted).
    */
-  public exportValue(exportedValue: any, options: ExportValueOptions = {}) {
+  public exportValue(exportedValue: any, options: ExportValueOptions = {}): any {
```

Comment:
Technically we could do better than `any`; the type is really `string | string[]`. `string | string[]` is a breaking change, so we would need a `exportListValue()` and an `exportValue()`. 

The user doesn't care if what they're exporting is a list or a string, they just want to export it. Having to decide which one to use doesn't provide any benefit, since we can figure out the type for them anyway.